exercises/move_semantics/move_semantics1.rs

```diff
@@ -3,19 +3,18 @@
 // Execute `rustlings hint move_semantics1` or use the `hint` watch subcommand
 // for a hint.
 
-// I AM NOT DONE
 
 #[test]
 fn main() {
     let vec0 = vec![22, 44, 66];
 
     let vec1 = fill_vec(vec0);
-
+    
     assert_eq!(vec1, vec![22, 44, 66, 88]);
 }
 
-fn fill_vec(mut vec: Vec<i32>) -> Vec<i32> {
```

Comment:
actually, I can leave this "mut" in the argument since the ownership is moved to the variable inside the function and only after that am I changing the value so it doesn't matter whether mut is there or not

Reply:
But why do we need a variable inside the function?